engine/class_modules/priest/sc_priest_shadow.cpp

```diff
@@ -1836,15 +1861,15 @@ struct dark_thoughts_t final : public priest_buff_t<buff_t>
     this->reactable = true;
   }
 
-  bool trigger( int stacks, double value, double chance, timespan_t duration ) override
+  /*bool trigger( int stacks, double value, double chance, timespan_t duration ) override
   {
     bool r = base_t::trigger( stacks, value, chance, duration );
 
     // Currently dark thoughts also resets your mind blast.
     priest().cooldowns.mind_blast->reset( true, 1 );
 
     return r;
-  }
+  }*/
```

Comment:
we can just remove this instead of commenting it out, the commit history will hold this if we need it back

Reply:
Very true, going to remove it.